### PR TITLE
feat(Illustration): add coffee illlustration

### PR DESCRIPTION
**Pull request by bndiep**
### Overview

Adds a coffee illustration.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/5rkfS3uWZsFNTsx2GmlDYA/Save-Feature-MVP?node-id=659%3A53606
- [x] Related to JIRA ticket: REACH-1950
- [x] I have run this code to verify it works

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.28.3-alpha.df097d.0<br/>@codecademy/gamut-kit@0.6.164-alpha.df097d.0<br/>@codecademy/gamut-labs@32.7.7-alpha.df097d.0<br/>@codecademy/gamut@48.3.0-alpha.df097d.0<br/>@codecademy/monaco@0.6.59-alpha.df097d.0<br/>@codecademy/styleguide@58.6.0-alpha.df097d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62d85a71641bc902fe3ff37e--gamut-preview.netlify.app>https://62d85a71641bc902fe3ff37e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62d85a71641bc902fe3ff37e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.28.3-alpha.1408db.0<br/>@codecademy/gamut-kit@0.6.164-alpha.1408db.0<br/>@codecademy/gamut-labs@32.7.7-alpha.1408db.0<br/>@codecademy/gamut@48.3.0-alpha.1408db.0<br/>@codecademy/monaco@0.6.59-alpha.1408db.0<br/>@codecademy/styleguide@58.6.0-alpha.1408db.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62d860de06a020087395c793--gamut-preview.netlify.app>https://62d860de06a020087395c793--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62d860de06a020087395c793>Deploy Logs</a></p>

File: packages/gamut-illustrations/src/Coffee.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <use xlinkHref="#c" transform="scale(.06778 .07118)" />
+      </pattern>
+      <image
+        id="c"
+        width="8"
+        height="8"
+        xlinkHref="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAICAYAAADED76LAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAeSURBVHgBzc4xAQAAAAEwEVxq6J+OGCzBQDk4MzgV8D0CqU26KA8AAAAASUVORK5CYII="
+      />
+    </defs>
+  </svg>
```
Comment (BandanaKM) [inline packages/gamut-illustrations/src/Coffee.tsx:60]: @bndiep Just one question - did we run this illustration through svgo before adding it to this component?

Comment (bndiep) [reply packages/gamut-illustrations/src/Coffee.tsx:60]: Yes! I used the link provided in gamut: https://jakearchibald.github.io/svgomg to optimize the svg.

I took a look to compare and I think the main differences are that the path drawn has been reduced and the ids are shortened to one character ("#a" vs. "#pattern0").

**Review by BandanaKM — APPROVED** [review]

File: packages/gamut-illustrations/src/Coffee.tsx (hunk 1 above)
Comment (BandanaKM) [reply packages/gamut-illustrations/src/Coffee.tsx:60]: Awesome :)

**Pull request merged**